.gitlab-ci.yml

```diff
@@ -319,7 +319,7 @@ build:edge+flambda:
   variables:
     OPAM_SWITCH: edge
     OPAM_VARIANT: "+flambda"
-    COQ_EXTRA_CONF: "-native-compiler yes -coqide opt -flambda-opts "
+    COQ_EXTRA_CONF: "-native-compiler no -coqide opt -flambda-opts "
```

Comment:
just a minor question: would
```suggestion
    COQ_EXTRA_CONF: "-native-compiler ondemand -coqide opt -flambda-opts "
```
be better? (to have a bit more backward-compat.) or maybe this question is irrelevant and the current version of the PR is very fine!

Reply:
Indeed, it could work this way.

Reply:
@proux01 , your opinion on this comment?

Reply:
That would require an overlay for bignum, because bignum is expecting the stdlib to be precompiled (unless native compiler has been disabled, in which case we only get a warning). So I'd rather fix bignum once this is merged.